apriltag_ros/src/continuous_detector.cpp

```diff
@@ -195,16 +202,22 @@ void ContinuousDetector::imageCallback (
   // compute the pixel values for the FOV pixel buffer (ie. resizing the FOV)
   fov_pixel_buffer_height_ = (camera_info->height/2) - ((camera_info->height * fov_size_scaler_) / 2);
   fov_pixel_buffer_width_ = (camera_info->width/2) - ((camera_info->width * fov_size_scaler_) / 2);
+  bool detector_validations_failed{ false };
 
   // Check if not in fov or not in detection range and publish an empty detection message
   if (!isTagInFOV(tag_pose_) || !isTagInDetectionRange(tag_pose_))
   {
-    // ROS_INFO_STREAM_THROTTLE(1.0, "FOV or range check not passed");
-    publishEmptyDetection(image_rect->header);
-    return;
+    if (detection_timeout_)
+    {
+      // ROS_INFO_STREAM_THROTTLE(1.0, "FOV or range check not passed");
+      publishEmptyDetection(image_rect->header);
+      return;
+    }
+    
+    detector_validations_failed = true;
   }
 
-  // ROS_INFO_STREAM_THROTTLE(0.1, "Frame " << image_rect->header.frame_id << " is in the FOV, detecting");
+  // ROS_INFO_STREAM_THROTTLE(0.1, "Frame " << image_rect->header.frame_id << " is in the FOV (or within time buffer), detecting");
```

Comment:
Remove or make DEBUG if too spammy

Reply:
Yea I removed in the previous MR (Nolan suggested it, the changes just haven't been reflected here yet).